config/manifests/bases/backstage-operator.clusterserviceversion.yaml

```diff
@@ -3,7 +3,8 @@ kind: ClusterServiceVersion
 metadata:
   annotations:
     alm-examples: '[]'
-    capabilities: Basic Install
+    capabilities: Seamless Upgrades
+    operatorframework.io/suggested-namespace: backstage-system
   name: backstage-operator.v0.0.0
```

Comment:
why do we have TWO versions of the csv file in this repo? manifests/bases/ and bundle/manifests/ ?

Reply:
The one in config/manifests is used as a basic configuration to generate the one under bundle/manifests.